geany.appdata.xml.in

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <_name>Geany IDE</_name>
+  <project_license>GPL-2.0+</project_license>
+  <_summary>A fast and lightweight IDE using GTK+</_summary>
+  <url type="homepage">http://geany.org/</url>
+  <description>
+    <_p>Geany is a small and lightweight integrated development environment.
+It was developed to provide a small and fast IDE, which has only a
+few dependencies from other packages. Another goal was to be as independent
+as possible from a special Desktop Environment like KDE or GNOME. So it
+is using only the GTK+ toolkit and therefore you need only the
+GTK+ runtime libraries to run Geany.</_p>
+    <ul>
+      <_li>syntax highlighting</_li>
+      <_li>code completion</_li>
+      <_li>auto completion of often used constructs like if, for and while</_li>
```

Comment:
Well… that's a question of how it reads, because "often used" is somewhat more true, it doesn't have to specifically be language constructs (this is talking about snippets)